projects/kit/utils/mask/create-date-mask.ts

```diff
@@ -0,0 +1,20 @@
+import {tuiAssert, TuiDateMode} from '@taiga-ui/cdk';
+import {TUI_DIGIT_REGEXP, TuiTextMaskList} from '@taiga-ui/core';
+
+export function tuiCreateDateMask(mode: TuiDateMode, separator: string): TuiTextMaskList {
+    tuiAssert.assert(separator.length === 1, 'Separator should consist of only 1 symbol');
+
+    const DAY = [TUI_DIGIT_REGEXP, TUI_DIGIT_REGEXP];
```

Comment:
nit: `Array(2).fill(TUI_DIGIT_REGEXP)`. Do you think we also need both `DAY` and `MONTH`? You can move them outside this function, btw. Also we use caps for constants outside function scopes anyway :)